code.js

```diff
@@ -17,7 +17,7 @@ function binarySearchRecursion(list, element, left, right){
         return binarySearchRecursion( list, element, left, mid - 1); //Also used ChatGPT here for the same airhead mistake
     }
 }
-
+/*
```

Comment:
After "if (list[mid] === element){" you need to check for duplication, too. So, you need another if statement after that before returning mid. Something like this: "if (mid === 0 || list[mid - 1] !== element) { return mid;"

Reply:
Thank you